ui/ui-frontend/projects/archive-search/src/app/archive/criteria-search/criteria-search.component.html

```diff
@@ -242,7 +242,7 @@
                       | translate)
                     : criteriaVal.key === 'ALL_ARCHIVE_UNIT_TYPES'
                       ? criteriaValue.label
-                      : criteriaValue.value.value
+                      : (getCriteriaLabel(criteriaVal.key, criteriaValue) | truncate: 50)
```

Comment:
Ne pas tronquer dans le tooltip

```suggestion
                      : getCriteriaLabel(criteriaVal.key, criteriaValue
```